### PR TITLE
Release 0.8

### PR DESCRIPTION
**Pull request by krl**
### Changed
- Change microkelvin and canonical versions to 0.6 and 0.7 respectively

Resolves: #24, #26

### CODE REVIEW

**Comment by krl** [comment]
fixes #24, #26

File: documentation/announcement.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Announcing nstack 0.8
```
Comment (ZER0) [inline documentation/announcement.md:1]: You can safely delete the whole `documentation` folder (see the wiki)

**Review by ZER0 — CHANGES_REQUESTED** [review]
Few things to fix before merging into the default branch.

A couple of things not mentioned in the comments:

1.  Add a proper description for the PR (you can basically copy the changelog in the description, plus adding "Resolves: xx" where xx is the issue is resolving.
2. Run clippy and ensure is not emit any warnings (either allowing something or fixing something. If you allow something a comment about why should be added)

**Review by ZER0 — APPROVED** [review]
LGTM!

**Pull request merged**